### PR TITLE
Create a recommdation for Raleigh

### PR DESCRIPTION
**Pull request by yinglimao**
This to Do:
- [ ] Add commit
- [ ] Review comment
- [ ] official review
- [ ] Merge

### CODE REVIEW

**Comment by kraghuram** [comment]
👍

**Comment by cr-tadd** [comment]
Comment from Tadd

File: yinglimao-raleigh.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Raleigh, NC
+
+### Places to Eat
+- City Tacos
+- Slater's 50/50
+- Sublime
```
Comment (cr-tadd) [inline yinglimao-raleigh.md:6]: Sublime is the name of a place?

File: yinglimao-raleigh.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+### Places to Visit
+- Coronado Island
+- Balboa Park
+
+### Places to Avoid
+- East County
```
Comment (kraghuram) [inline yinglimao-raleigh.md:13]: 👍

**Review by kraghuram — APPROVED** [review]
👍

**Comment by cr-tadd** [comment]
Good

**Pull request merged**